process-document/src/main/java/com/ritense/processdocument/service/impl/CamundaProcessJsonSchemaDocumentService.java

```diff
@@ -330,11 +332,15 @@ public StartProcessForDocumentResult startProcessForDocument(StartProcessForDocu
     }
 
     public Document getDocument(DelegateExecution execution) {
+        denyAuthorization();
         return getDocument(ProcessInstanceId.fromExecution(execution, CamundaProcessInstanceId.class), execution);
```

Comment:
This method should probably also have a deny?

Reply:
So this particular method should probably be deprecated, as this can only really be called from inside a process, but since this isn't a process bean that's not really possible. I'll add a note about this particular method to the process beans story, so it'll be deprecated then.